### PR TITLE
Fixed Swedish Translations

### PR DESCRIPTION
**Pull request by BronzW**

### CODE REVIEW

File: kibbeh/public/locales/sv/translation.json [hunk 1]
```diff
@@ -62,17 +67,17 @@
...
       "deleteAccount": "Radera konto",
-      "overlaySettings": "Gå till overlayinställningar",
+      "overlaySettings": "Overlayinställningar",
       "couldNotFindUser": "Sorry, we could not find that user"
     },
     "notFound": {
       "whoopsError": "Whoops! Denna sida blev förlorad i konversationen.",
       "goHomeMessage": "Ingen fara. Du kan",
-      "goHomeLinkText": "gå hem"
+      "goHomeLinkText": "gå hem."
```
Comment (goldyydev) [inline kibbeh/public/locales/sv/translation.json]: Why is the end period needed?

File: kibbeh/public/locales/sv/translation.json [hunk 2]
```diff
@@ -274,6 +287,8 @@
       "galacticDoge": "Galaktiskdoge",
       "spottedLife": "Planet med liv upptäckt"
     },
-    "feed": { "yourFeed": "Your Feed" }
+    "feed": {
+      "yourFeed": "Ditt flöde"
+    }
   }
-}
+}
```
Comment (goldyydev) [inline kibbeh/public/locales/sv/translation.json]: No newline
```suggestion
}

```

**Review by goldyydev — CHANGES_REQUESTED** [review]

File: kibbeh/public/locales/sv/translation.json (hunk 1 above)
Comment (goldyydev) [reply kibbeh/public/locales/sv/translation.json]: ~~Also, no comma~~

File: kibbeh/public/locales/sv/translation.json [hunk 3]
```diff
@@ -274,6 +287,8 @@
       "galacticDoge": "Galaktiskdoge",
       "spottedLife": "Planet med liv upptäckt"
     },
-    "feed": { "yourFeed": "Your Feed" }
+    "feed": {
+      "yourFeed": "Ditt flöde"
+    }
   }
 }
```
Comment (HarrisonMayotte) [inline kibbeh/public/locales/sv/translation.json:294]: New line is missing:
```suggestion
}

```

**Review by HarrisonMayotte — CHANGES_REQUESTED** [review]

File: kibbeh/public/locales/sv/translation.json (hunk 3 above)
Comment (BronzW) [reply kibbeh/public/locales/sv/translation.json:294]: Didn't I fix this?

Comment (HarrisonMayotte) [reply kibbeh/public/locales/sv/translation.json:294]: It did not appear to be resolved. I just realized I can commit this myself so I'll do that then merge.

**Pull request merged**